From f1a24198af04ad3d0ebc9dffde454beb19ed0804 Mon Sep 17 00:00:00 2001
From: Andreas Jaeger <aj@suse.de>
Date: Mon, 29 Apr 2013 21:11:13 +0200
Subject: [PATCH 3/4] BZ#15085: Fix comments/strings for RES_NOCHECKNAME

	[BZ #15085]
	* resolv/resolv.h (RES_NOCHECKNAME): Mark as unimplemented.
	* resolv/res_debug.c (p_option): Mark RES_NOCHECKNAME as
	unimplemented.
---
 ChangeLog          | 7 ++++++-
 NEWS               | 8 ++++----
 resolv/res_debug.c | 2 +-
 resolv/resolv.h    | 2 +-
 4 files changed, 12 insertions(+), 7 deletions(-)

diff --git a/resolv/res_debug.c b/resolv/res_debug.c
index c064d0c..c1d8c02 100644
--- a/resolv/res_debug.c
+++ b/resolv/res_debug.c
@@ -585,7 +585,7 @@ p_option(u_long option) {
 	case RES_NOALIASES:	return "noaliases";
 	case RES_USE_INET6:	return "inet6";
 	case RES_ROTATE:	return "rotate";
-	case RES_NOCHECKNAME:	return "no-check-names";
+	case RES_NOCHECKNAME:	return "no-check-names(unimpl)";
 	case RES_KEEPTSIG:	return "keeptsig(unimpl)";
 	case RES_BLAST:		return "blast";
 	case RES_USEBSTRING:	return "ip6-bytstring";
diff --git a/resolv/resolv.h b/resolv/resolv.h
index ed15a70..53c3bba 100644
--- a/resolv/resolv.h
+++ b/resolv/resolv.h
@@ -207,7 +207,7 @@ struct res_sym {
 #define	RES_NOALIASES	0x00001000	/* shuts off HOSTALIASES feature */
 #define	RES_USE_INET6	0x00002000	/* use/map IPv6 in gethostbyname() */
 #define RES_ROTATE	0x00004000	/* rotate ns list after each query */
-#define	RES_NOCHECKNAME	0x00008000	/* do not check names for sanity. */
+#define	RES_NOCHECKNAME	0x00008000	/* do not check names for sanity (!IMPL) */
 #define	RES_KEEPTSIG	0x00010000	/* do not strip TSIG records */
 #define	RES_BLAST	0x00020000	/* blast all recursive servers */
 #define RES_USEBSTRING	0x00040000	/* IPv6 reverse lookup with byte
-- 
1.8.2.3

